README.md

```diff
@@ -2,4 +2,37 @@
 
 ![](https://github.com/qiime2/q2-composition/workflows/ci/badge.svg)
 
-This is a QIIME 2 plugin. For details on QIIME 2, see https://qiime2.org.
\ No newline at end of file
+This is a QIIME 2 plugin. For details on QIIME 2, see https://qiime2.org.
+
+## Install Guide
+
+This plugin is a wrapper for the ANCOMBC package. We are working with the
+[RELEASE_3_15](https://github.com/FrederickHuangLin/ANCOMBC/tree/RELEASE_3_15)
+branch, as this is where the functionality for ANCOM, ANCOM-BC, and SECOM are
+all located. This release hasn't been published on bioconductor yet, so the
```

Comment:
`This bioconductor release is not yet available on bioconda` it is 100% available on bioconductor, it just isn't compatible with our older R (because we're on the last bioconductor release still)

Reply:
Whoops, thank you - I was wondering if I had mixed those up!